backend/entityservice/views/run/description.py

```diff
@@ -23,21 +25,33 @@ def get(project_id, run_id):
     return RunDescription().dump(run_object)
 
 
+def _is_postgres_integrity(exc):
+    return isinstance(exc, psycopg2.IntegrityError)
+
+
+@retry(wait_exponential_multiplier=1000, wait_exponential_max=10000, retry_on_exception=_is_postgres_integrity)
```

Comment:
this could potentially go on forever. Be a good parent and set a limit.

Reply:
In the docs it said it would go up to the `wait_exponential_max` seconds. Same with the version of moved to from the tenacity docs